### PR TITLE
Add doc for observation properties support

### PR DESCRIPTION
**Pull request by kmoscoe**
Also add a troubleshooting item for failed schema checks.

### CODE REVIEW

File: custom_dc/custom_data.md [hunk 1]
```diff
@@ -180,6 +186,16 @@ The first set of parameters only applies to `foo.csv`. The second set of paramet
 
 You must specify the provenance details under `sources`.`provenances`; this field associates one of the provenances defined there to this file.
 
+`observationProperties`
+
+: Optional: Additional information about the observations contained in the CSV file. Currently, four properties are supported:
```
Comment (keyurva) [inline custom_dc/custom_data.md:191]: Consider `s/about the observations/about each observation`

File: custom_dc/custom_data.md [hunk 2]
```diff
@@ -180,6 +186,16 @@ The first set of parameters only applies to `foo.csv`. The second set of paramet
...
+- `unit`: The unit of measurement used in the observations. This is a string representing a currency, area, weight, volume, etc. For example, `SquareFoot`, `USD`, `Barrel`, etc.
```
Comment (keyurva) [inline custom_dc/custom_data.md:192]: Consider linking to the corresponding glossary term for each property in [here](https://docs.datacommons.org/glossary.html#scaling-factor).

File: custom_dc/troubleshooting.md [hunk 3]
```diff
@@ -43,6 +43,13 @@ Failed to create metadata: failed to create secret manager client: google: could
 
 This indicates that you have not specified API keys in the environment file. Follow procedures in [One-time setup steps](/custom_dc/quickstart.html#setup) to obtain and configure API keys.
 
+### "SQL schema check failed"
+
+This error indicates that there is a problem with the database schema. Check for the following additional errors:
+
+- "The following columns are missing..." -- This indicates that there is has been an update to the schema in the latest Docker build. To remedy this, restart the data management Docker container and then try again to start the services container.
```
Comment (keyurva) [inline custom_dc/troubleshooting.md:50]: Perhaps, re<i>run</i> the data management docker container?

Also, "restart the services container" vs "try again to start the services container"?

File: custom_dc/troubleshooting.md [hunk 4]
```diff
@@ -43,6 +43,13 @@ Failed to create metadata: failed to create secret manager client: google: could
...
+- "Cannot create tables in database" -- This indicates a problem with your CSV files and/or config.json file. Please see [Prepare and load your own data](/custom_d/custom_data.html) for details.
```
Comment (keyurva) [inline custom_dc/troubleshooting.md:51]: Hmm - this does not indicate a problem with their CSV or json file. Let's chat about this error offline and add a separate item for it. For the time being, let's remove this.

**Review by keyurva — COMMENTED** [review]
Thanks for the super quick turnaround!

File: custom_dc/custom_data.md (hunk 1 above)
Comment (kmoscoe) [reply custom_dc/custom_data.md:191]: Done.

File: custom_dc/custom_data.md (hunk 2 above)
Comment (kmoscoe) [reply custom_dc/custom_data.md:192]: Good idea. Done.

File: custom_dc/troubleshooting.md (hunk 3 above)
Comment (kmoscoe) [reply custom_dc/troubleshooting.md:50]: Done.

File: custom_dc/troubleshooting.md (hunk 4 above)
Comment (kmoscoe) [reply custom_dc/troubleshooting.md:51]: OK

File: custom_dc/troubleshooting.md [hunk 5]
```diff
@@ -45,10 +45,10 @@ This indicates that you have not specified API keys in the environment file. Fol
 
 ### "SQL schema check failed"
 
-This error indicates that there is a problem with the database schema. Check for the following additional errors:
+This error indicates that there is a problem with the database schema. Check for the following additional error:
+
+- "The following columns are missing..." -- This indicates that there is has been an update to the schema in the latest Docker build. To remedy this, rerun the data management Docker container and then restart the services container.
```
Comment (keyurva) [inline custom_dc/troubleshooting.md:50]: nit: Perhaps we can drop the docker build part and just say "This indicates that there is has been an update to the database schema. To remedy this..."

**Review by keyurva — APPROVED** [review]

File: custom_dc/troubleshooting.md (hunk 5 above)
Comment (kmoscoe) [reply custom_dc/troubleshooting.md:50]: OK

**Pull request merged**